go/libraries/doltcore/env/actions/remotes.go

```diff
@@ -191,6 +191,9 @@ func PushToRemoteBranch(ctx context.Context, rsr env.RepoStateReader, tempTableD
 	wg, progChan, pullerEventCh := progStarter(newCtx)
 	err = Push(ctx, tempTableDir, mode, destRef.(ref.BranchRef), remoteRef.(ref.RemoteRef), localDB, remoteDB, cm, progChan, pullerEventCh)
 	progStopper(cancelFunc, wg, progChan, pullerEventCh)
+	if err == nil {
```

Comment:
Instead of separate logic, you should handle this as a case of the switch below (and remove the `err != nil` check above the switch):

```go
switch err {
    case nil:
        cli.PrintLn()
        return nil
}
```